client_is_human.js

```diff
@@ -0,0 +1,23 @@
+var DALVIK_DETECT = new RegExp('^Dalvik/');
```

Comment:
You can start using ES6 now in api.

If you don't want to / would rather stick with the old style, we should have a conversation about it as a team, because if we're not all pulling in the same direction our code is going to get pretty weird pretty quickly. I'd rather not have half of writing ES6 and the other half not.